app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -584,19 +584,21 @@ export class AppCommandParser {
         const result: AutocompleteSuggestion[] = [];
 
         const bindings = this.getCommandBindings();
+
         for (const binding of bindings) {
-            let base = binding.app_id;
+            let base = binding.label;
             if (!base) {
                 continue;
             }
 
             if (base[0] !== '/') {
                 base = '/' + base;
             }
+
             if (base.startsWith(command)) {
                 result.push({
+                    Complete: binding.label,
                     Suggestion: base,
-                    Complete: base.substring(1),
```

Comment:
Same comment as the webapp PR https://github.com/mattermost/mattermost-webapp/pull/7823#discussion_r608145357

> Do we want to support the scenario where an app has put a slash in front of the label? This code will no longer support this because of this change ☝️

Reply:
See https://github.com/mattermost/mattermost-webapp/pull/7823#discussion_r608404129